### ISSUE TITLE
Feature: Include markdown tables

### ISSUE THREAD

**Issue by ArnyminerZ** [ArnyminerZ/MarkdownText#36]
Made by @tysino at TechbeeAT/jtxBoard#146
---
**Is your feature request related to a problem? Please describe.**

Weirdly, table syntax is recognized, but the formatting characters (|, -) are just removed in the rendered markdown.

**Describe the solution you'd like**

Implement github-style rendered tables (maybe just a library change is necessary).

**Describe alternatives you've considered**
Other option: show the formatting characters in the rendered markdown (which is ugly, but at least not unexpected).

**Additional context**
[The Android client for Nextcloud notes](https://github.com/stefan-niedermann/nextcloud-notes) supports this, if you are looking for a library.

PS: Thank you for the awesome app.

**Comment by ArnyminerZ** [comment]
As a temporal fix, I have made (#40) that the table is rendered as code:
![image](https://user-images.githubusercontent.com/12086466/212124673-7f74d6ee-d5e3-49e7-a4ca-bea860c3e5cb.png)

Once I have more free time I will try to implement a proper table renderization feature.

**Comment by ArnyminerZ** [comment]
I've managed to draw some raw content into an image that kind of looks like a table. I think rendering an image and loading it will be the best method with our current implementation.

With my current algorithm, with the following code:
```kotlin
Image(
    bitmap = TableRenderer(
         listOf("Column 1", "Column 2", "Column 3"),
         listOf(
             listOf("Value 1", "Value 2", "Value 3"),
             listOf("Value 4", "Value 5", "Value 6"),
             listOf("Value 7", "Value 8", "Value 9"),
        )
    ).imageBitmap,
    contentDescription = "",
    modifier = Modifier.fillMaxWidth()
)
```
Renders this:

![image](https://user-images.githubusercontent.com/12086466/212168823-de31b40b-42d2-451d-8428-741cc7f6d61b.png)

**Comment by graue70** [comment]
If you render the table as an image, that means you would lose the ability to select (in order to copy) text from inside the table, right? Is that worth it?
I think I prefer the current raw table solution over an image.

**Comment by tysino** [comment]
> If you render the table as an image, that means you would lose the ability to select (in order to copy) text from inside the table, right? Is that worth it? I think I prefer the current raw table solution over an image.

Yes I agree, a complete rendering of a table with selectable text would be better.
